pkg/engine/engine_test.go

```diff
@@ -587,3 +587,14 @@ func TestGameEngineSendBatchReturnsExecutionPanicErrorOnPanicEverywhere(t *testi
 	}
 	engine.Close()
 }
+
+func TestGenerateSeededGame(t *testing.T) {
```

Comment:
What does this test?

Reply:
Now compares the game's deck and seeded deck